### PR TITLE
remove old weather modules

### PR DESCRIPTION
**Pull request by khassel**
see https://github.com/MichMich/MagicMirror/pull/2914

~~This PR should be merged with the next mm release Oct. 2022, not before.~~
I think we can merge this before next release because the old modules are already only dummies so the old documentation makes no sense ...

Hopefully I found all occurencies of the old modules ...

### CODE REVIEW

**Review by KristjanESPERANTO — APPROVED** [review]
Looks good :+1:

File: modules/compliments.md [hunk 1]
```diff
@@ -51,25 +51,6 @@ The `compliments` property contains an object with at least four arrays: <code>m
 
 Compliments can be set for a specific day in the format `YYYY-MM-DD`. `.` can be used as a wildcard.
 
-If set, the currentweather can be used for compliments. The available properties are:
```
[the remark this answers is not in the document]
Comment (khassel) [reply modules/compliments.md:54]: you know or you think ;) What about the properties, are they identical?

[the remark this answers is not in the document]
Comment (rejas) [reply modules/compliments.md:54]: It still does, sicne I am using it :-) The properties are identical

[the remark this answers is not in the document]
Comment (khassel) [reply modules/compliments.md:54]: fixed ...

**Review by rejas — APPROVED** [review]

**Pull request merged**